src/gas-hub.js

```diff
@@ -298,32 +292,25 @@ function initPageEvent() {
   });
 }
 
+function auth() {
+  return new Promise((resolve, reject) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/fd132db2233355d7308db18ded515a069ff50e80/src/gas-hub.js#L296"></span>‘reject’ is defined but never used.</td>
    </tr>
  </tbody>
</table>